test/src/org/labkey/test/tests/targetedms/TargetedMSQCConfigureMetricTest.java

```diff
@@ -0,0 +1,86 @@
+package org.labkey.test.tests.targetedms;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.components.targetedms.ConfigureQCMetricsWebPart;
+import org.labkey.test.components.targetedms.QCSummaryWebPart;
+import org.labkey.test.pages.targetedms.PanoramaDashboard;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Category({})
+@BaseWebDriverTest.ClassTimeout(minutes = 2)
+public class TargetedMSQCConfigureMetricTest extends TargetedMSTest
```

Comment:
If this test extends `TargetedMSPremiumTest` instead, it would have access to some QC metric related methods.